files/en-us/web/api/globaleventhandlers/ondragenter/index.html

```diff
@@ -28,89 +28,109 @@ <h3 id="Return_value">Return value</h3>
- ev.preventDefault();
+  event_log.textContent += "dragOver\n";
+  // Change the target element's border to signify a drag over event
+  // has occurred
+  ev.currentTarget.style.background = "lightblue";
+  ev.preventDefault();
 }
 
 function drop_handler(ev) {
- console.log("Drop");
- ev.preventDefault();
- var data = ev.dataTransfer.getData("text");
- ev.target.appendChild(document.getElementById(data));
+  event_log.textContent += "Drop\n";
+  ev.preventDefault();
+  const data = ev.dataTransfer.getData("text");
+  ev.target.appendChild(document.getElementById(data));
 }
 
 function dragenter_handler(ev) {
- console.log("dragEnter");
- // Change the source element's background color for enter events
- ev.currentTarget.style.background = "yellow";
+  event_log.textContent += "dragEnter\n";
+  // Change the source element's background color for enter events
+  ev.currentTarget.style.background = "yellow";
 }
 
 function dragleave_handler(ev) {
- console.log("dragLeave");
- // Change the source element's border back to white
- ev.currentTarget.style.background = "white";
+  event_log.textContent += "dragLeave\n";
+  // Change the source element's border back to white
+  ev.currentTarget.style.background = "white";
 }
 
 function dragend_handler(ev) {
- console.log("dragEnd");
- // Change the target element's background color to visually indicate
- // the drag ended.
- var el=document.getElementById("target");
- el.style.background = "pink";
+  event_log.textContent += "dragEnd\n";
+  // Change the target element's background color to visually indicate
+  // the drag ended.
+  target.style.background = "pink";
 }
 
-function init() {
- // Set handlers for the source's enter/leave/end/exit events
- var el=document.getElementById("source");
- el.ondragenter = dragenter_handler;
- el.ondragleave = dragleave_handler;
- el.ondragend = dragend_handler;
-}
-&lt;/script&gt;
-&lt;body onload="init();"&gt;
-&lt;h1&gt;Examples of &lt;code&gt;ondragenter&lt;/code&gt;, &lt;code&gt;ondragleave&lt;/code&gt;, &lt;code&gt;ondragend&lt;/code&gt;&lt;/h1&gt;
- &lt;div&gt;
-   &lt;p id="source" ondragstart="dragstart_handler(event);" draggable="true"&gt;
-     Select this element, drag it to the Drop Zone and then release the selection to move the element.&lt;/p&gt;
- &lt;/div&gt;
- &lt;div id="target" ondrop="drop_handler(event);" ondragover="dragover_handler(event);"&gt;Drop Zone&lt;/div&gt;
-&lt;/body&gt;
-&lt;/html&gt;
+// Set handlers for the source's drag - start/enter/leave/end events
+source.ondragstart = dragstart_handler;
+source.ondragenter = dragenter_handler;
+source.ondragleave = dragleave_handler;
+source.ondragend = dragend_handler;
+
+// Set handlers for the target's drop and dragover events
+target.ondrop = drop_handler;
+target.ondragover = dragover_handler;
+
+// Set click event listener on button to reload the example
+const button = document.querySelector("#reload");
+button.addEventListener("click", () => {
+    document.location.reload();
```

Comment:
```suggestion
  document.location.reload();
```

Consistent 2-apace indentation.